src/controllers/employer.ts

```diff
@@ -91,3 +91,26 @@ export const createJobPost = async (
     next(new InternalServerError(error.message))
   }
 }
+
+export const deleteJobPostbyId = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const id = parseInt(req.params.id)
+    const isJobFound = JobPost.findOne(id)
+    if (!isJobFound) {
+      return next(new NotFoundError('Job is no more available'))
+    }
+    await getConnection()
+      .createQueryBuilder()
+      .delete()
+      .from(JobPost)
+      .where('id = :id', { id: 1 })
```

Comment:
I believe you just need to call remove on the found post, ex: `foundPost.remove( )`